ImageProvider.py

```diff
@@ -115,7 +115,15 @@ def _imgProcess(q_out, config, video_source, video_file, stopped, started):
         # Open video stream
         if video_source == None:
             logger.info("Starting from video file input: %s" % (video_file,))
-            _videoStream = cv2.VideoCapture(video_file)
+            # use HW acceleration for video file
+            if get_config("USE_GSTREAM"):
+                _videoStream = cv2.VideoCapture('filesrc location={}\
+                                        ! queue ! h264parse ! omxh264dec ! nvvidconv \
+                                        ! video/x-raw,format=BGRx,width=960,height=544 ! queue ! videoconvert ! queue \
```

Comment:
The ImageProvider uses a resolution of either 960x540 or 1920x1080 did it also work with a height of 544 when you tested it and is it also possbile to get images with 540 in height? A resizing step could then be dropped.

Reply:
Hello Fabian,  yes, it is possible to resize video on the flight by using GStreamer with hw NVDEC it should be very fast. I recorded the video with running application and jtop utility. In the left bottom corner you can ses that NVDEC is on. 

https://user-images.githubusercontent.com/19520821/179599264-d1805b82-116c-4e2d-8666-a09859b23e34.mov

It is possible because openCV compiled with CUDA. You can see this on screenshot below. I suggest to use image from source that I mentioned in pull request.

![Compiled_with_CUDA](https://user-images.githubusercontent.com/19520821/179601003-99f07d6e-e249-42ec-a4fc-7f13f2e9b7b7.png)

And I overclocked my jetson nano to 2Ghz CPU and 998Mhz GPU. I use cooling tower from aliexpress, see below:  
 
![IMG_5242](https://user-images.githubusercontent.com/19520821/179601679-2df5432d-349a-436d-995f-2547173cf9a8.jpg)